app/controllers/sessions_controller.rb

```diff
@@ -6,6 +6,7 @@ def create
     user = User.find_by email: params[:session][:email].downcase
     if user&.authenticate(params[:session][:password])
```

Comment:
user&.authenticate(params[:session][:password])
tại sao ở đây em sử dụng toán tử & 
Nó khác gì với user.authenticate(params[:session][:password])

Reply:
Về toán tử && sẽ return về 1 giá true nếu cả hai toán hạng trái phải đều true, nếu không là false cho các trường hợp khác. 
Ở đây, em sử dụng toán tử & để viết rút gọn câu lệnh : user && user.authenticate(params[:session][:password]).